src/earn/EarnEnterAmount.tsx

```diff
@@ -669,6 +777,12 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'flex-start',
   },
+  lineItem: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
```

Comment:
I think we want `flex-start` here in case the fee amount overflows (similar to `txDetailsLineItem`)

<img src="https://github.com/user-attachments/assets/72dfdc95-55da-4144-8be5-6b9cd2bbe027" width="250" />